docs/diagrams/SortCommand.puml

```diff
@@ -60,4 +60,4 @@ Command <|-- SortCommand
 SortCommand --> CommandResult
 SortCommand --> Model
 
-@enduml
\ No newline at end of file
+@enduml
```

Comment:
My mistake! Good catch on your part, appreciate your hard work and cooperation. Thanks!